commands/faas.go

```diff
@@ -56,6 +57,11 @@ func Execute(customArgs []string) {
 	faasCmd.SilenceErrors = true
 	faasCmd.SetArgs(customArgs[1:])
 	if err := faasCmd.Execute(); err != nil {
+		if len(err.Error()) > 2 {
```

Comment:
Not sure I like this, are you setting this to 2 to protect against the SplitAfterN not returning 2 elements?

How about this:

```
func capitaliseFirst(str string) string {
    for _, c := range str {
        return string(unicode.ToUpper(c)) + str[1:]
    }  
    return ""
}
```

You can then just:
```
fmt.Println(capitaliseFirst(err.Error())
```